scripts/upgrade-via-npm.ps1

```
@@ -96,365 +166,801 @@ if (-not $CMD) {
-# [1/3] Download and extract new version
-Write-Host "[1/5] Downloading new version..."
-$TMPDIR_PACK = Join-Path ([System.IO.Path]::GetTempPath()) "qqbot-pack-$([guid]::NewGuid().ToString('N').Substring(0,8))"
-$EXTRACT_DIR = Join-Path ([System.IO.Path]::GetTempPath()) "qqbot-extract-$([guid]::NewGuid().ToString('N').Substring(0,8))"
-New-Item -ItemType Directory -Path $TMPDIR_PACK -Force | Out-Null
-New-Item -ItemType Directory -Path $EXTRACT_DIR -Force | Out-Null
+$CONFIG_SNAPSHOT_FILE = ""
+$PREV_RELOAD_MODE = ""
 
-try {
-    Push-Location $TMPDIR_PACK
+function Snapshot-Config {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Snapshot-Config' uses an unapproved verb.

(PSUseApprovedVerbs)